RecursiveExtractor/FileEntry.cs

```diff
@@ -179,10 +208,24 @@ public static async Task<FileEntry> FromStreamAsync(string name, Stream content,
             {
                 FullPath = $"{parent?.FullPath}{Path.DirectorySeparatorChar}{name}";
             }
+            Stream Content = new MemoryStream();
+
+            try
+            {
+                if (content.Length > memoryStreamCutoff)
+                {
+                    Content = new FileStream(Path.GetTempFileName(), FileMode.Create, FileAccess.ReadWrite, FileShare.ReadWrite, 4096, FileOptions.DeleteOnClose);
```

Comment:
Suggest replacing the magic 4096 with a constant.

Reply:
Done.